scripts/ngi_pipeline_start.py

```diff
@@ -217,82 +138,38 @@ def __init__(self, *args, **kwargs):
-                args.restart_finished_jobs = \
-                    validate_dangerous_user_thing(action=("restart FINISHED jobs, deleting "
-                                                          "previous analysis files"))
-            if args.__dict__.get("restart_running_jobs"):
-                args.restart_finished_jobs = \
-                    validate_dangerous_user_thing(action=("restart RUNNING jobs, deleting "
-                                                          "previous analysis files"))
+        if args.__dict__.get("restart_failed_jobs"):
+            args.restart_failed_jobs = \
+                validate_dangerous_user_thing(action=("restart FAILED jobs, deleting "
+                                                        "previous analysies files"))
+        if args.__dict__.get("restart_finished_jobs"):
+            args.restart_finished_jobs = \
+                validate_dangerous_user_thing(action=("restart FINISHED jobs, deleting "
+                                                        "previous analysis files"))
+        if args.__dict__.get("restart_running_jobs"):
+            args.restart_finished_jobs = \
+                validate_dangerous_user_thing(action=("restart RUNNING jobs, deleting "
+                                                        "previous analysis files"))
     # Charon-specific arguments ('organize', 'analyze', 'qc')
     if args.__dict__.get("force_update"):
         args.force_update = \
                 validate_dangerous_user_thing("overwrite existing data in Charon")
-    if args.__dict__.get("delete_existing"):
-        args.delete_existing = \
-                validate_dangerous_user_thing("delete existing data in Charon")
-
 
     # Finally execute corresponding functions
 
-    ## Analyze Flowcell
-    if 'analyze_fc_dirs' in args:
-        LOG.info('Starting flowcell analysis of flowcell {} '
-                 '{}'.format(inflector.plural("directory", len(args.analyze_fc_dirs)),
-                             ", ".join(args.analyze_fc_dirs)))
-        flowcell.process_demultiplexed_flowcells(args.analyze_fc_dirs,
-                                                 args.restrict_to_projects,
-                                                 args.restrict_to_samples,
-                                                 args.restart_failed_jobs,
-                                                 args.restart_finished_jobs,
-                                                 args.restart_running_jobs,
-                                                 keep_existing_data=args.keep_existing_data,
-                                                 no_qc=args.no_qc,
-                                                 quiet=args.quiet,
-                                                 manual=True,
-                                                 generate_bqsr_bam=args.generate_bqsr_bam)
-
     ## Analyze Project
     elif 'analyze_project_dirs' in args:
```

Comment:
```suggestion
    if 'analyze_project_dirs' in args:
```
Looks like the initial `if` was removed?

Reply:
Ah, you're right! Well spotted 😄